ChaosMod/Effects/db/Screen/ScreenBouncyRadar.cpp

```diff
@@ -0,0 +1,64 @@
+#include "Memory/UI.h"
+#include <stdafx.h>
```

Comment:
```suggestion
#include <stdafx.h>

#include "Memory/UI.h"
```
stdafx.h should always be the first included header in source files, you can add an empty line between both to stop clang-format from reshuffling those